prov/efa/src/rdm/rxr_rma.c

```diff
@@ -307,7 +311,46 @@ ssize_t rxr_rma_read(struct fid_ep *ep, void *buf, size_t len, void *desc,
 	return rxr_rma_readv(ep, &iov, &desc, 1, src_addr, addr, key, context);
 }
 
-/* rma_write functions */
+/**
+ * @brief Decide if we should issue this WRITE using rdma-core, or via emulation.
+ *
+ * @param ep		The endpoint.
+ * @param tx_entry	The op_entry context for this write.
+ * @param peer		The peer we will be writing to.
```

Comment:
nit: can we add the `[in]`/`[out]` annotations to the params here (and elsewhere)?